### PR TITLE
chore: update `dotnet` instr. to command-line

### PR DESCRIPTION
**Pull request by luizhlelis**
**Description:** <Describe what has changed.>

change `dotnet` instrumentation into command line

**Link to tracking Issue(s):** <Issue number if applicable>

- Resolves: #2671

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: .chloggen/dotnet-instrumentation-command-flag.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```
Comment (yuriolisa) [inline .chloggen/dotnet-instrumentation-command-flag.yaml:2]: Please set it as an enhancement.

File: main.go [hunk 2]
```diff
@@ -190,6 +193,7 @@ func main() {
 		config.WithCreateRBACPermissions(createRBACPermissions),
 		config.WithEnableMultiInstrumentation(enableMultiInstrumentation),
 		config.WithEnableApacheHttpdInstrumentation(enableApacheHttpdInstrumentation),
+		config.WithEnableDotnetInstrumentation(enableDotnetInstrumentation),
```
Comment (yuriolisa) [inline main.go:196]: Pls use capital N.

File: main.go [hunk 3]
```diff
@@ -172,6 +174,7 @@ func main() {
 		"labels-filter", labelsFilter,
 		"enable-multi-instrumentation", enableMultiInstrumentation,
 		"enable-apache-httpd-instrumentation", enableApacheHttpdInstrumentation,
+		"enable-dotnet-instrumentation", enableDotnetInstrumentation,
```
Comment (yuriolisa) [inline main.go:177]: Here also.

**Review by yuriolisa — CHANGES_REQUESTED** [review]
Just small changes to do. Do you mind adding e2e-tests for auto-instrumentations cli commands?

File: .chloggen/dotnet-instrumentation-command-flag.yaml (hunk 1 above)
Comment (swiatekm-sumo) [reply .chloggen/dotnet-instrumentation-command-flag.yaml:2]: Why should it be an enhancement? We're removing a beta feature flag that users might have set. That's a breaking change imo, and that's how we've treated similar changes in the past.

File: main.go (hunk 2 above)
Comment (luizhlelis) [reply main.go]: done ✅

File: main.go (hunk 3 above)
Comment (luizhlelis) [reply main.go]: done ✅ 
I've just updated all the references that I worked on

**Review by yuriolisa — APPROVED** [review]

**Review by swiatekm-sumo — APPROVED** [review]

**Comment by jaronoff97** [comment]
Thank you for your contribution!

**Pull request merged**